.github/workflows/test.yml

```diff
@@ -36,7 +36,10 @@ jobs:
           restore-keys: |
             cache-v1-
 
-      - name: Compile and Test
+      - name: Compile
```

Comment:
I'm just curious - is this done to make it fail faster if code doesn't compile in one of the scala versions?

Reply:
At first, it was an attempt to avoid **Travis** to kill the job because a single step took more than 40 minutes.

Then, I kind of liked the idea of making sure it compiled before running tests. Because I sometimes forget to run `+compile` or `+test` in local and finding that out faster seemed like a good idea.

But I am not too attached to it.

Reply:
No, it makes sense. Let's leave it this way.